internal/app/agent/service.go

```diff
@@ -168,41 +177,76 @@ func (a *agent) Control(uuid, cmdStr string) error {
 		return err
 	}
 
-	payload, err := encodeSenML(uuid, cmd, resp)
-	if err != nil {
-		return err
-	}
-
-	if err := a.Publish(a.config.Agent.Channels.Control, string(payload)); err != nil {
-		return err
-	}
-
-	return nil
+	return a.processResponse(uuid, cmd, resp)
 }
 
 // Message for this command
-// "[{"bn":"1:", "n":"config", "vs":"export, /configs/export/config.toml, config_file_content"}]"
+// [{"bn":"1:", "n":"services", "vs":"view"}]
+// [{"bn":"1:", "n":"config", "vs":"save, export, filename, filecontent"}]
 // config_file_content is base64 encoded marshaled structure representing service conf
 // Example of creation:
```

Comment:
You can write code samples in a separate file. Godoc will recognize `ExampleServerConfig` and create a nice `example` section. You can take a look here https://pkg.go.dev/fmt?tab=doc#Sprintf, and the actual code in `fmt` source file `example_test.go`

Reply:
I think that we should create issue to write tests for this application

Reply:
https://github.com/mainflux/agent/issues/22